.circleci/config.yml

```diff
@@ -79,6 +104,35 @@ jobs:
             - ./dist/*.whl
             - ./dist/*.tar.gz
 
+  integration-test-spark:
+    working_directory: ~/openlineage/integration/spark
+    machine: true
+    environment:
+      TESTCONTAINERS_RYUK_DISABLED: "true"
+      JDK8_HOME: /usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
We've been meaning to change `JDK8_HOME` env var to `JAVA8_HOME`. Mind renaming as part of this PR?

Reply:
Actually, let's just leave it as is to avoid breaking dev environments.